schema-guide.md

```diff
@@ -1776,1070 +2231,1287 @@ An entity can represent different types of entities, such as a team, an institut
         type: generic
     ```
 
 ### `definitions.reference.loc-end`
 
-- **type**: `...`
+- **type**: `integer`
 - **required**: `false`
-- **description**: ...
+- **description**: The line of code in the file where the work ends.
 - **usage**:<br><br>
     ```yaml
     preferred-citation:
-      loc-end: ...
+      loc-end: 42
       type: generic
     ```
     ```yaml
     references:
-      - loc-end: ...
+      - loc-end: 42
         type: generic
     ```
 
 ### `definitions.reference.loc-start`
 
-- **type**: `...`
+- **type**: `integer`
 - **required**: `false`
-- **description**: ...
+- **description**: The line of code in the file where the work starts.
 - **usage**:<br><br>
     ```yaml
     preferred-citation:
-      loc-start: ...
+      loc-start: 21
       type: generic
     ```
     ```yaml
     references:
-      - loc-start: ...
+      - loc-start: 21
         type: generic
     ```
 
 ### `definitions.reference.location`
 
-- **type**: `...`
+- **type**: [`definitions.entity`](#definitionsentity)
 - **required**: `false`
-- **description**: ...
+- **description**: The location of the work.
 - **usage**:<br><br>
     ```yaml
     preferred-citation:
-      location: ...
+      location:
+        name: "Library of the Unseen University"
       type: generic
     ```
     ```yaml
     references:
-      - location: ...
+      - location:
+          name: "Library of the Unseen University"
         type: generic
     ```
 
 ### `definitions.reference.medium`
 
-- **type**: `...`
+- **type**: Nonempty `string`
 - **required**: `false`
-- **description**: ...
+- **description**: The medium of the work.
 - **usage**:<br><br>
     ```yaml
     preferred-citation:
-      medium: ...
+      medium: "datasette"
       type: generic
     ```
     ```yaml
     references:
-      - medium: ...
+      - medium: "datasette"
         type: generic
     ```
 
 ### `definitions.reference.month`
 
-- **type**: `...`
+- **type**: `integer` in range `1-12`
 - **required**: `false`
-- **description**: ...
+- **description**: The month in which a work has been published.
 - **usage**:<br><br>
     ```yaml
     preferred-citation:
-      month: ...
+      month: 7
       type: generic
     ```
     ```yaml
     references:
-      - month: ...
+      - month: 7
         type: generic
     ```
 
 ### `definitions.reference.nihmsid`
 
-- **type**: `...`
+- **type**: Nonempty `string`
 - **required**: `false`
-- **description**: ...
+- **description**: The NIHMSID of a work.
 - **usage**:<br><br>
     ```yaml
     preferred-citation:
-      nihmsid: ...
+      nihmsid: "NIHMS236863"
       type: generic
     ```
     ```yaml
     references:
-      - nihmsid: ...
+      - nihmsid: "NIHMS236863"
         type: generic
     ```
 
 ### `definitions.reference.notes`
 
-- **type**: `...`
+- **type**: Nonempty `string`
 - **required**: `false`
-- **description**: ...
+- **description**: Notes pertaining to the work.
```

Comment:
Should we mention that notes are not supposed to be picked up by downstream tooling (if that's even true)?

Reply:
Good idea.

Reply:
Good point! I don't think it's true for all tooling (reference managers may want to pick this up!), but some. Added https://github.com/citation-file-format/citation-file-format/pull/262/files#diff-28af06810cb7c393bb9ba0787f608e4f97918cc6b304541ba867916f243b4a38R3070-R3071. What do you think?